niri-ipc/src/lib.rs

```diff
@@ -129,6 +131,10 @@ pub enum Response {
     FocusedOutput(Option<Output>),
     /// Information about the focused window.
     FocusedWindow(Option<Window>),
+    /// Information about the picked window.
+    ///
+    /// `None` if the request was cancelled by the user.
```

Comment:
I guess there's other ways to get None, like clicking on empty space or a layer surface or whatever. But I also guess this can be through of as cancelled?

What I'm getting at is maybe this clarification in the comment isn't needed altogether since it kinda makes it read like only explicit cancelling with Escape will return None?